contributors/design-proposals/pod-preset.md

```diff
@@ -210,6 +212,10 @@ This will be first implemented as an AdmissionControl plug-in then can be
 converted to an Initializer once that is fully ready. The proposal for
 Initializers can be found at [kubernetes/community#132](https://github.com/kubernetes/community/pull/132).
 
+#### PodPresetOptOut Annotation
+There may be instances where you wish for a pod to not be altered by any pod
+preset mutations. For these events, one can add an annotation in the pod spec
+of the form: `podpreset.admission.kubernetes.io/PodPresetOptOut: "true"`.
```

Comment:
the annotation was changed in review to: "podpreset.admission.kubernetes.io/exclude"